using_packages/conanfile_txt.rst

```diff
@@ -41,12 +41,12 @@ Here is an example:
 .. code-block:: text
 
     [requires]
-    Poco/1.9.0@pocoproject/stable
+    Poco/1.9.4@pocoproject/stable
```

Comment:
Why leaving the user/channel here?

Reply:
because there is an explanation about each section in the package reference. To avoid any confusion, I've renamed that example to `mypackage`